mediator/src/main/scala/io/iohk/atala/mediator/db/BsonImplicits.scala

```diff
@@ -1,16 +1,16 @@
-package fmgp.did.db
+package io.iohk.atala.mediator.db
 
-import scala.util._
-import zio.json._
-import reactivemongo.api.bson._
+import fmgp.crypto.*
+import fmgp.did.*
+import fmgp.did.comm.*
+import fmgp.util.*
+import reactivemongo.api.bson.*
+import zio.json.*
 
-import fmgp.crypto._
-import fmgp.did._
-import fmgp.did.comm._
-import fmgp.util._
+import scala.util.*
 
 given BSONWriter[DIDSubject] with {
-  import DIDSubject._
+  import DIDSubject.*
```

Comment:
Can you tell me again why we replace that `_` with `*`?

Reply:
will fix it may I did it by mistake